raft.c

```diff
@@ -149,11 +152,13 @@ static void executeRaftRedisCommandArray(RaftRedisCommandArray *array,
             continue;
         }
 
+        uint64_t begin_rmcall = timeMonotonicNanos();
         enterRedisModuleCall();
         RedisModuleCallReply *reply = RedisModule_Call(
                 ctx, cmd, redis_raft.resp_call_fmt, &c->argv[1], c->argc - 1);
         int ret_errno = errno;
         exitRedisModuleCall();
+        RedisModule_LatencyAddSample("rm_call", (mstime_t) ((timeMonotonicNanos() - begin_rmcall) / 1000000));
```

Comment:
Looks like `RedisModule_LatencyAddSample()` has millisecond precision only, so maybe we just use RedisModule_Milliseconds(). It has less precise timer call (gettimeofday(), less precise, better for performance).